tests/system/providers/amazon/aws/example_batch.py

```diff
@@ -250,6 +264,7 @@ def delete_job_queue(job_queue_name):
         submit_batch_job,
         wait_for_batch_job,
         # TEST TEARDOWN
+        job,
```

Comment:
You can just call describe_job here like we do with the others below to make things a little cleaner, right?
```suggestion
        describe_job(submit_batch_job.output),
```